src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, ensemble: Ensemble, config: EvaluatorServerConfig):
             list[tuple[EVENT_HANDLER, Event]]
         ] = asyncio.Queue()
         self._max_batch_size: int = 500
-        self._batching_interval: float = 2.0
+        self._batching_interval: float = 0.5
```

Comment:
I thought about this, but the issue arises when having max running more than 200.  Have you tried big poly with this?

Reply:
I will try with bigpoly, but which issue or negative impact would you look for?

Reply:
Performance wise. That is whether it makes gui more or less responsive, since this gives more work to ee.

Reply:
When you flood the evaluator with messages (as in bigpoly), the parameter ` _max_batch_size` will determine the behaviour (as far as I read the code), not the batching interval.